### ISSUE TITLE
"Don't take foreground window on Window servers"

### ISSUE THREAD

**Issue by johnny-mac** [debauchee/barrier#47]
### Operating Systems ###
Artix Linux

Server: Artix & Void

Client: Artix & Devuan

### Barrier Version ###

2.1.1 

### Steps to reproduce bug ###

"Don't take foreground window on Window servers"

I'm sorry if I misunderstand this but shouldn't this option stop barrier from making my game window become 'not the main window' when I move my mouse to the other screen? Its really annoying, I'd like to use this while gaming & watching videos but this messes my game up & sometimes my PS3 controller will stop working.... I take my mouse off my main PC to my secondary one & it makes it not the main window, the active window... but its not like I clicked on anything else on that window, I just moved my mouse to the other one... Apparently this works for some people but I can't seem to make it work for me.

**Comment by walker0643** [comment]
Not quite, @johnny-mac. Your mouse never really leaves your primary screen in the technical sense... the same way a magician's fingers never leave his hands... part of the magic of Barrier is that it simulates a single mouse being connected to and communicating with multiple machines while only being physically connected to one of them.

TLDR: When your mouse is active on another screen it can't also be active inside your game window on the primary screen. Same with the keyboard.
